app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  DEFAULT_RENDERER_PAGINATE = WillPaginate::ActionView::Bootstrap4LinkRenderer
+
+  def will_paginate(collection_or_options = nil, options = {})
+    if collection_or_options.is_a? Hash
```

Comment:
@sinpai if a method is called after some operator, then let's add parentheses
```ruby
if collection_or_options.is_a?(Hash)
```
```ruby
options = options.merge(renderer: DEFAULT_RENDERER_PAGINATE) unless options[:renderer]
```

Reply:
@sinpai will be great if you'll find a rubocop metric for that and write to `#general` slack chat for other students because it's usual mistake

Reply:
Task accepted. :relaxed:

Reply:
cop is Lint/RequireParentheses

Reply:
Thanks!